sonar-db/src/main/java/org/sonar/db/purge/PurgeDao.java

```diff
@@ -130,15 +132,13 @@ private static void purge(ResourceDto project, String[] scopesWithoutHistoricalD
 
   private void disableOrphanResources(final ResourceDto project, final SqlSession session, final PurgeMapper purgeMapper, final PurgeListener purgeListener) {
     final List<IdUuidPair> componentIdUuids = new ArrayList<>();
-    session.select("org.sonar.db.purge.PurgeMapper.selectComponentIdUuidsToDisable", project.getId(), new ResultHandler() {
-      @Override
-      public void handleResult(ResultContext resultContext) {
+    session.select("org.sonar.db.purge.PurgeMapper.selectComponentIdUuidsToDisable", project.getId(),
+      resultContext -> {
         IdUuidPair componentIdUuid = (IdUuidPair) resultContext.getResultObject();
         if (componentIdUuid.getId() != null) {
           componentIdUuids.add(componentIdUuid);
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Make this line start at column 9. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://dory.sonarsource.com/sonarqube/coding_rules#rule_key=squid%3AIndentationCheck)